dev/source/docs/using-gazebo-simulator-with-sitl.rst

```diff
@@ -0,0 +1,205 @@
+.. _using-gazebo-simulator-with-sitl:
+
+================================
+Using Gazebo Simulator with SITL
+================================
+
+This article explains how to use Gazebo http://gazebosim.org/
+as an external simulator for Copter.
+
+Overview
+========
+
+Gazebo is a well-known and respected robotics simulator. We will be compiling 
+Gazebo from source, because no current release has built-in support for ArduCopter.
+
+.. warning::
+   Gazebo support is still under work on October 2016. The following instructions may be outdated or thing could be broken. In those case, open an issue on ardupilot github or discuss.
+
+.. tip::
+
+   Gazebo is particularly useful for defining autonomous
+   indoor flights or swarms.
+
+
+Preconditions
+=============
+
+We recommend Ubuntu starting from 14.04.2 (it is working on 16.04 but untest on 16.10) as this is the platform used for testing
+this approach and is also known to be compatible with SITL.
+
+Compiling and installing Gazebo
+===============================
+
+We will be using a standard version of ArduPilot, but a custom fork of Gazebo until gazebo plugin get merge into Gazebo build.
```

Comment:
"We will be using a standard version of ArduPilot but a custom fork of Gazebo, until the gazebo plugin gets merge into Gazebo-master."